test/functional/connection/connection.ts

```diff
@@ -1,10 +1,16 @@
 import "reflect-metadata";
 import {expect} from "chai";
 import {Post} from "./entity/Post";
+import {Guest as GuestV1} from "./entity/v1/Guest";
+import {Comment as CommentV1} from "./entity/v1/Comment";
+import {Guest as GuestV2} from "./entity/v1/Guest";
+import {Comment as CommentV2} from "./entity/v1/Comment";
```

Comment:
are you sure? Can you check once again? Because what you did is same as:

```ts
const a = {};
const b = a;
```

You have same instances of GuestV1 and GuestV2, CommentV1 and CommentV2